.github/workflows/build-and-test.yml

```diff
@@ -107,6 +107,12 @@ jobs:
           target: wasm32-unknown-unknown
           default: true
 
+      - name: Install protobuf
+        run: |
+          sudo apt-get update && \
+          sudo apt-get upgrade -y && \
+          sudo apt-get install -yq openssl clang libclang-dev cmake  protobuf-compiler    
```

Comment:
Can you please remove the `upgrade -y` line and only install the needed deps? The step name is "install protobuf" but you are installing a lot more than just protobuf

Reply:
Thanks for pointing out the deficiencies in the code, for this issue I will open an issue specifically to fix it